src/protean/server/__init__.py

```diff
@@ -1,3 +1,13 @@
+"""
+Protean Server module
+"""
+
 from .engine import Engine
 
-__all__ = ["Engine"]
+try:
+    from protean.server.fastapi_server import FastAPIServer
+
+    __all__ = ["Engine", "FastAPIServer"]
```

Comment:
We don't want to expose `FastAPIServer` like this. Eventually, the implementation will be abstracted because we should retain the freedom to switch out FastAPI and replace it with something else.